resources/management/commands/assign_area_regions.py

```diff
@@ -0,0 +1,114 @@
+
+from resources.enums import AreaRegion
+from resources.models import Harbor, WinterStorageArea
+
+HARBOR_REGIONS = {
+    AreaRegion.EAST: [
+        "Airorannan venesatama",
+        "Aurinkolahden venesatama (Aurinkoranta)",
+        "Aurinkolahden venesatama (Pursilahdenranta)",
+        "Aurinkolahden venesatama (Pursilahdenranta)",
+        "Honkaluoto / Venesatama",
+        "Hopeasalmen venesatama",
+        "Kipparlahden venesatama",
+        "Koivuniemen venesatama",
+        "Laivalahden venesatama",
+        "Meri-Rastilan venesatama",
+        "Mustikkamaan venesatama",
+        "Nandelstadhin venesatama",
+        "Naurissalmen venesatama (Kipparlahdensilmukka 9)",
+        "Pikku Kallahden venesatama",
+        "Porslahden venesatama",
+        "Puotilan venesatama",
+        "Sarvaston venesatama",
+        "Strömsinlahden venesatama",
+        "Vasikkasaaren venesatama",
+        "Vuosaarenlahden venesatama",
+        "Vähäniityn venesatama",
+    ],
+    AreaRegion.WEST: [
+        "Eläintarhanlahden venesatama (Pitkänsillanranta)",
+        "Eläintarhanlahden venesatama",
+        "Hietalahdenallas / Venesatama",
+        "Katajanokan venesatama",
+        "Lähteelä retkisatama",
+        "Merihaan venesatama (Hakaniemenranta 13)",
+        "Merisatama (Ehrenströmintie 21)",
+        "Merisatama (Merisatama 28)",
+        "Merisatama (Merisatamanranta 20)",
+        "Pajalahden venesatama (Meripuistotie) / Venesatama",
+        "Pohjoisranta 3 / Venesatama",
+        "Ramsaynrannan venesatama (Ramsaynranta 4)",
+        "Ruoholahden venesatama (Jaalaranta)",
+        "Ruoholahden venesatama (Kellosaarenranta)",
+        "Ruoholahden venesatama (Ruoholahden kanava)",
+        "Salmisaaren venesatama",
+        "Saukonpaaden venesatama",
+        "Saunalahden venesatama (Ramsaynranta 6)",
+        "Siltavuoren venesatama",
+        "Tammasaaren allas",
+        "Tervasaari (Tervasaarenkannas 1) / Venesatama",
+    ],
+}
+
+WINTER_STORAGE_AREA_REGIONS = {
+    AreaRegion.EAST: [
+        "Marjaniemi",
+        "Puotila",
+        "Ruusuniemi I",
+        "Ruusuniemi II",
+        "Strömsinlahti",
+    ],
+    AreaRegion.WEST: ["Kaisaniemi", "Hernesaari", "Lähteelä", "Pajalahti"],
+}
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Assigning Harbor regions")
+        self._assign_area_regions(Harbor, HARBOR_REGIONS)
+        self.stdout.write("-----")
+        self.stdout.write("Assigning Winter Storage Area regions")
+        self._assign_area_regions(WinterStorageArea, WINTER_STORAGE_AREA_REGIONS)
+
+    def _assign_area_regions(self, model, regions):
+        successful = {}
+        total_successful = 0
+        failed = []
+
+        for region, areas in regions.items():
+            successful[region.label] = 0
+            for area_name in areas:
+                try:
+                    area = model.objects.translated(name=area_name).first()
```

Comment:
isn't it safer to specify the `"fi"` lang code here?

Reply:
🤔 yeah, good catch